.travis.yml

```diff
@@ -6,8 +6,16 @@ php:
   - hhvm
   - hhvm-nightly
 
+env:
+  global:
+    - GIT_NAME:  "'Marco Pivetta'"
+    - GIT_EMAIL: ocramius@gmail.com
+    - GH_REF:    github.com/Ocramius/ProxyManager
+    - secure:    "KEY"
```

Comment:
@malukenho please point me to any documentation where this key should be generated